services/frontend/src/scenes/ProjectDetails/ProjectDetails.scss

```diff
@@ -59,3 +59,34 @@
     }
 }
 
+#project-contributors{
+    .project-contributors-content{
+        display: grid;
```

Comment:
Another style thing: rules should be ordered alphabetically. I've been letting it slide, but since we don't have a css linter, this is your reminder.